src/js/providers/tracks-mixin.js

```diff
@@ -385,6 +385,10 @@ define(['../utils/underscore',
         return providerName.indexOf('flash') === -1 && (utils.isChrome() || utils.isIOS() || utils.isSafari());
     }
 
+    function _kindSupported(kind) {
```

Comment:
@hghazzi This checks whether if ```subtitles``` or ```captions``` is in the string, and I think Evol created this function to check the same thing as ```if (track.kind === 'subtitles' || track.kind === 'captions')```.